bpf/kmesh/workload/xdp.c

```diff
@@ -78,8 +78,8 @@ static inline int should_shutdown(struct xdp_info *info, struct bpf_sock_tuple *
 
 static inline int parser_xdp_info(struct xdp_md *ctx, struct xdp_info *info)
 {
-    void *begin = (void *)(ctx->data);
-    void *end = (void *)(ctx->data_end);
+    void *begin = (void *)(long)(ctx->data);
+    void *end = (void *)(long)(ctx->data_end);
```

Comment:
what is the size of long? Or it depends? 

Can you explicitly replace with a fixed type

Reply:
This is just a type conversion to clear the alarm

Reply:
I mean can you replace long with u32

Reply:
sometimes sizeof(long)=64